lxd/db/cluster/update.go

```diff
@@ -63,6 +63,13 @@ var updates = map[int]schema.Update{
 	24: updateFromV23,
 	25: updateFromV24,
 	26: updateFromV25,
+	27: updateFromV26,
```

Comment:
This needs to be rebase and become FromV27 following the recent update from @freeekanayaka